fluid-project/infusion-docs
# Migrate to 11ty.

This Pull Request migrates the SSG from Docpad to 11ty.

To test locally: 

- run `npm run 11ty` and open up the local server as pointed by the console output.

Some notes:

- To see a specific version, just put the version name after the baseURL like so: http://<localhost>/infusion/development/**v1.5.0**/ComponentOptionsAndDefaults.html
- The tests don't work anymore. Since they were from docpad era, they need to be migrated to 11ty.
